airflow/providers/opsgenie/provider.yaml

```diff
@@ -19,7 +19,7 @@
 package-name: apache-airflow-providers-opsgenie
 name: Opsgenie
 description: |
-    `Opsgenie <https://www.opsgenie.com/>`__
+    `Opsgenie <https://www.atlassian.com/software/opsgenie>`__
```

Comment:
Not related to this PR, but seems like we might want to move this provider into the `atlassian.opsgenie`

Reply:
Yeah. We should. We already moved 'jira' to 'atlassian-jira' so we could follow that one.